prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -441,7 +442,18 @@ class Hl7Serializer(val metadata: Metadata) {
 
         terser.set(formPathSpec("OBX-11", aoeRep), "F")
         terser.set(formPathSpec("OBX-14", aoeRep), date)
+        terser.set(formPathSpec("OBX-23-7", aoeRep), "XX")
```

Comment:
This just specifies that the ID type for the testing lab is of type `organization identifier`: https://hl7-definition.caristix.com/v2/HL7v2.5.1/Tables/0203

This value is apparently required.

Reply:
Hmm. If we set OBX-23-7 Identifier ID type, don't we need to include OBX-23-3 Identifier ID?